config/nrfconnect/args.gni

```diff
@@ -28,3 +29,13 @@ chip_system_project_config_include = ""
 
 custom_toolchain = "//toolchain:zephyr"
 mbedtls_target = "//:dummy"
+
+pw_log_BACKEND = "$dir_pw_log_basic"
```

Comment:
I guess we should somehow make these parameters optional. Ideally they should be provided by the app since pw_sys_io is not part of CHIP, but temporarily we may go with `if (chip_build_pw_rpc_lib) ...`, too

Reply:
Ok, for now moved I it to the pw_rpc.gni file to keep args.gni clean, but anyway still it would be good to import it conditionally.